third_party/2and3/pymysql/connections.pyi

```diff
@@ -92,6 +92,7 @@ class Connection:
     def close(self): ...
     def autocommit(self, value): ...
     def commit(self): ...
+    def begin(self): ...
```

Comment:
Can you add a `-> None` return type? (I know all the functions around it don't have it either, but let's set higher standards in new code.)

Reply:
@JelleZijlstra definitely can add the `-> None` annotation. That said, if you'd like me to add `-> None` to the other methods as well, I would be happy to do so. Just let me know.

Reply:
Sorry, scratch that. Just took a deeper look and realised that the `self._read_ok_packet()
 ` call at the end of `begin(self)` returns [`ok`](https://github.com/PyMySQL/PyMySQL/blob/6e5f4ef33923ec9988d35ae815ecda9a54629564/pymysql/connections.py#L773), which is an `OKPacketWrapper`.

I get the feeling that the other methods also might not returning `None`, so I'll leave them be for now.

Reply:
But (at least from the code you posted above), it doesn't return the return value of `_read_ok_packet`. Python doesn't have an implicit return value.

Reply:
@JelleZijlstra You are absolutely correct (can you tell Python isn't my strong suite? ;) ). Fixed by going back to your original recommendation. Sorry about that.